src/families/bitcoin/getAccountNetworkInfo.ts

```diff
@@ -47,10 +47,12 @@ export async function getAccountNetworkInfo(
   }
   // Fix fees if suggested fee is too low, this is only for viacoin/decred because the fees backend endpoint is broken
   if (
-    (account.currency.id === "viacoin" || account.currency.id === "decred") &&
+    (account.currency.id === "viacoin" ||
+      account.currency.id === "decred" ||
+      account.currency.id === "qtum") &&
```

Comment:
Not for this PR, but in the future we could consider changing this kind of hardcoded test to use the coin-specific mechanisms that we have here: https://github.com/LedgerHQ/ledger-live-common/blob/develop/src/families/bitcoin/logic.ts#L138